internal/armada/server.go

```diff
@@ -196,6 +198,27 @@ func Serve(config *configuration.ArmadaConfig, healthChecks *health.MultiChecker
 		}
 		submitServerToRegister = pulsarSubmitServer
 
+		// If postgres details were provided, enable deduplication.
+		log.Infof("DedupTable: %s\n", config.Pulsar.DedupTable)
+		log.Infof("Postgres: %+v\n", config.Pulsar.Postgres)
```

Comment:
be careful here else you might end up logging postgres passwords.

Reply:
You're right; removed.